Mathlib/Topology/ContinuousFunction/CompactlySupported.lean

```diff
@@ -0,0 +1,709 @@
+/-
+Copyright (c) 2024 Yoh Tanimoto. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yoh Tanimoto
+-/
+import Mathlib.Topology.ContinuousFunction.Bounded
+import Mathlib.Topology.ContinuousFunction.CocompactMap
+import Mathlib.Topology.ContinuousFunction.Compact
+import Mathlib.Topology.ContinuousFunction.ZeroAtInfty
+
+/-!
+# Compactly supported continuous functions
+
+The type of compactly supported continuous functions. When the domain is compact,
+`C(α, β) ≃ C_c(α, β)` via the identity map. When the codomain is a metric space, every continuous
+compactly supported map is a bounded continuous function. When the domain is a locally
+compact space, this type has nice properties.
+
+## TODO
+
+* Create more intances of algebraic structures (e.g., `NonUnitalSemiring`) once the necessary
+  type classes (e.g., `TopologicalRing`) are sufficiently generalized.
+-/
+
+
+universe u v w
+
+variable {F : Type*} {α : Type u} {β : Type v} {γ : Type w} [TopologicalSpace α]
```

Comment:
Most of the time, it's not needed to specify universes in Lean, unless in very specific situations. So I'd just go for
```suggestion
variable {F α β γ : Type*} [TopologicalSpace α]
```